sg_jira/handlers/entity_issue_handler.py

```diff
@@ -931,22 +939,12 @@ def _get_shotgun_assignment_from_jira_issue_change(
                             # iterating
                             break
             if to_assignee:
-                # Try to add the new assignee to the Shotgun assignment
-                # Use the Issue assignee value to avoid a Jira user query
                 jira_user = jira_issue["fields"]["assignee"]
-                sg_user = self._shotgun.find_one(
-                    "HumanUser",
-                    [["email", "is", jira_user["emailAddress"]]],
-                    ["email", "name"]
+                sg_user = self._jira_user_to_shotgun(
```

Comment:
When looking at the `to` field (`if to_assignee`), we're passing in the id and the assignee field from JIRA. For JIRA server, that's an optmization because the email can be found directly in the assignee field.